desktop/src/main/java/bisq/desktop/main/support/dispute/agent/DisputeAgentView.java

```diff
@@ -54,7 +50,7 @@ public DisputeAgentView(DisputeManager<? extends DisputeList<? extends DisputeLi
                             ContractWindow contractWindow,
                             TradeDetailsWindow tradeDetailsWindow,
                             AccountAgeWitnessService accountAgeWitnessService,
-                            @Named(AppOptionKeys.USE_DEV_PRIVILEGE_KEYS) boolean useDevPrivilegeKeys) {
```

Comment:
It's not obvious to me why injection of this value would still work without the explicit annotation. The value of `AppOptionKeys.USE_DEV_PRIVILEGE_KEYS` is in fact "useDevPrivilegeKeys", spelled exactly the same as the `useDevPrivilegeKeys` parameter, so in theory Guice could introspect and do the injection based on that, but this information is not inspectable at runtime unless the `-parameters` option has been passed to javac to keep the parameter symbol table available (and our build does not do this). What am I missing?

Reply:
The class is abstract and never instantiated directly. The annotation comes from a time before the type hierarchy was in place.

Reply:
The sub-classes should have the annotation.

Reply:
Roger that, and I see that the subclasses are in fact annotated. Thanks.

utACK